{
  level: 'error',
  message: 'UnCaught rejection is detected from serve.ts Operation `users.findOne()` buffering timed out after 10000ms',
  stack: 'MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n' +
    '    at Timeout.<anonymous> (C:\\course\\NEXT_LEVEL_2\\university-mangement\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:185:23)\n' +
    '    at listOnTimeout (node:internal/timers:559:17)\n' +
    '    at processTimers (node:internal/timers:502:7)',
  label: 'right meow!',
  timestamp: '2023-06-18T04:49:58.501Z'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 5000,
  level: 'error',
  message: 'UnCaught rejection is detected from serve.ts listen EADDRINUSE: address already in use :::5000',
  stack: 'Error: listen EADDRINUSE: address already in use :::5000\n' +
    '    at Server.setupListenHandle [as _listen2] (node:net:1432:16)\n' +
    '    at listenInCluster (node:net:1480:12)\n' +
    '    at Server.listen (node:net:1568:7)\n' +
    '    at Function.listen (C:\\course\\NEXT_LEVEL_2\\university-mangement\\node_modules\\express\\lib\\application.js:635:24)\n' +
    '    at C:\\course\\NEXT_LEVEL_2\\university-mangement\\src\\server.ts:24:18\n' +
    '    at Generator.next (<anonymous>)\n' +
    '    at fulfilled (C:\\course\\NEXT_LEVEL_2\\university-mangement\\src\\server.ts:5:58)\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:96:5)',
  label: 'right meow!',
  timestamp: '2023-06-18T04:53:36.052Z'
}
